generate-spec

```diff
@@ -43,6 +43,7 @@ $command
 	->option('--no-tags', 'Use no tags')
 	->option('--continue-on-error', 'Continue on error')
 	->option('--openapi-version', 'OpenAPI version to use', null, '3.0.3')
+	->option('--parameters-to-body', 'Put all parameters into the JSON body')
```

Comment:
When would someone do this? Who would need to do this?

Since it's in `generate-spec` I guess the maintainer of the API providing code, but it seems it's all-or-nothing for the whole app? Should I enable this for Talk?

Reply:
Good question, we should probably just switch all parameters into the body all the time so that the experience is always the same and not a mess of different locations for different parameters when calling the endpoint.

Reply:
I made a local test and worked fine to me.

Reply:
> OCI endpoint

You mean OCS?

> is necessary to have all parameters specified at requestBody because if not, the documentation will be wrong and will consider that all parameters is send by query string

No that is wrong, parameters can be either in the body or the query parameters. Even putting values in both for a single request works afaik.

> Only make sense to use this when is not a GET because by definition, GET haven't body.

No, GET requests can also have bodies, just some implementations can cause troubles which is not the case for Nextcloud/PHP (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET). For the sake of simplicity and consistency all parameters will just go into the body regardless of the HTTP method.

> I made a local test and worked fine to me when I used --parameters-to-body

What are you talking about? I know this code works (and the previous code also works just fine)

Reply:
> You mean OCS?

Sorry, is this, OCS. Was a typo.

> For the sake of simplicity and consistency all parameters will just go into the body regardless of the HTTP method.

 Ok

> What are you talking about? I know this code works (and the previous code also works just fine)

Sorry again by my comment at this PR. I'm only helping the code review making local tests.

Reply:
The flag is gone now and everything is moved to the body by default.